connectors/CompaniesHouse/apiProperties.json

```diff
@@ -0,0 +1,21 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "The Authorization for this api",
+          "displayName": "Authorization",
+          "tooltip": "Provide your Authorization"
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff"
```

Comment:
AFAIK, white color should not be used for a backgraound

Reply:
Changed to match their logo colour